manifests/master.pp

```diff
@@ -22,7 +22,7 @@
 #  ['puppet_ssldir']            - Puppet sll directory
 #  ['puppet_docroot']           - Doc root to be configured in apache vhost
 #  ['puppet_vardir']            - Vardir used by puppet
-#  ['puppet_passenger_port']    - Port to configure passenger on default 8140
+#  ['puppet_proxy_port']        - Port to configure the proxy on - default 8140
```

Comment:
I know why this has been done but it's a backwards breaking change so we are going to have to make this a major version release.